src/mscorlib/src/System/Threading/SpinLock.cs

```diff
@@ -77,13 +77,13 @@ public struct SpinLock
         private const int SPINNING_FACTOR = 100;
 
         // After how many yields, call Sleep(1)
-        private const int SLEEP_ONE_FREQUENCY = 40;
+        private const int SLEEP_ONE_FREQUENCY_MASK = 64;
 
         // After how many yields, call Sleep(0)
-        private const int SLEEP_ZERO_FREQUENCY = 10;
+        private const int SLEEP_ZERO_FREQUENCY_MASK = 16;
 
         // After how many yields, check the timeout
-        private const int TIMEOUT_CHECK_FREQUENCY = 10;
+        private const int TIMEOUT_CHECK_FREQUENCY_MASK = 16;
```

Comment:
I'm a little concerned about these changes.  I don't remember how much effort went into selecting the values initially, but these could have a real impact on usage, and issues that arise from such changes could be difficult to spot from limited microbenchmark-based testing.  Lots of factors impact this, including number of cores, layout of cores, usage patterns, etc.